transport/transport-backend-cct/src/test/java/com/google/android/datatransport/cct/CctTransportBackendTest.java

```diff
@@ -106,14 +109,16 @@ private BackendRequest getCCTBackendRequest(String transportName, CCTDestination
                         .setEventMillis(INITIAL_WALL_TIME)
                         .setUptimeMillis(INITIAL_UPTIME)
                         .setTransportName(transportName)
-                        .setPayload(PAYLOAD.toByteArray())
+                        .setEncodedPayload(
```

Comment:
Should we test the case in which the payload isn't encoded in a supported format?

Reply:
Done